Polyhedron/demo/Polyhedron/Plugins/PMP/Polyhedron_slicer_plugin.cpp

```diff
@@ -65,7 +65,7 @@ public Q_SLOTS:
   void on_Generate_button_clicked();
   bool on_Update_plane_button_clicked();
   void plane_manipulated_frame_modified();
-  void item_about_to_be_destroyed(Scene_item* scene_item);
+  void item_about_to_be_destroyed(CGAL::Three::Scene_item* scene_item);
   void dock_widget_closed();
```

Comment:
There is a `using namespace CGAL::Three;` at the beginning of `Polyhedron_slicer_plugin.cpp` (line 29), so calling `Scene_item*` directly works.